deploy_slave.sh

```diff
@@ -78,6 +78,9 @@ getent group piwheels || groupadd piwheels
 getent passwd piwheels || useradd -g piwheels -m -s /bin/bash piwheels
 passwd -d piwheels
 
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | CARGO_HOME=/usr/local sh -s -- -y
+echo "CARGO_HOME=/usr/local/bin" > /etc/environment.d/20-rustup.conf
```

Comment:
What about `/etc/profile.d`?

```suggestion
echo "export CARGO_HOME=/usr/local" > /etc/profile.d/rustup.sh
```

Reply:
`/etc/profile.d` is only loaded on interactive login session, not in scripts etc.

Reply:
```bash
touch /home/piwheels/.bashrc  # make sure .bashrc file exists
echo 'export CARGO_HOME=/usr/local' >> /home/piwheels/.bashrc
```

Might be safer to append to `.bashrc`?

Reply:
`.bashrc` usually contains non-critical tasks for interactive shells, like setting console aliases (which have not effect in scripts), colouring console outputs like `ls`. However, `bash` itself calls it on non-interactive shells/scripts as well. For this reason, it sometimes contains an exit path for non-interactive shells, which would then break the export when appended.

For "safety" it could be reviewed what the file contains on this build system, but it cannot contain anything important, also because the `piwheels` user is freshly created so that at best the skeletton dir default `.bashrc` is used.